### DISCUSSION TITLE
returns invalid JSON array "coordinates"

### DISCUSSION THREAD

**Discussion by joepperkins** [weather-gov/api#279] [Q&A]
**Example Endpoint:** https://api.weather.gov/gridpoints/HGX/64,92

[[[...]]] is not a valid JSON array structure for coordinates and is breaking all endpoints that include "coordinates"

{
    "@context": [
        "https://geojson.org/geojson-ld/geojson-context.jsonld",
        {
            "@version": "1.1",
            "wx": "https://api.weather.gov/ontology#",
            "geo": "http://www.opengis.net/ont/geosparql#",
            "unit": "http://codes.wmo.int/common/unit/",
            "@vocab": "https://api.weather.gov/ontology#"
        }
    ],
    "type": "Feature",
    "geometry": {
        "type": "Polygon",
        **"coordinates": [
            [
                [
                    -95.380197499999994,
                    29.6908821
                ],
                [
                    -95.380123999999995,
                    29.6681214
                ],
                [
                    -95.353927299999995,
                    29.668182999999999
                ],
                [
                    -95.353995699999999,
                    29.690943699999998
                ],
                [
                    -95.380197499999994,
                    29.6908821
                ]
            ]
        ]**
    },
    "properties": ...

**Comment by StephenClouse** [comment]
I have validated this through geojsonlint and this is valid GeoJSON. I have also tested with [geojson.io](http://geojson.io/#id=gist:StephenClouse/0f7fbf8294e4c4af409d9021d00cfb94&map=15/29.6795/-95.3671).

A Polygon is an array of one or more linear rings (arrays) containing arrays of coordinate pairs.

Please clarify what you believe is invalid here.
